src/libcore/intrinsics.rs

```diff
@@ -1452,6 +1452,9 @@ fn overlaps<T>(src: *const T, dst: *const T, count: usize) -> bool {
 ///   size_of::<T>()` bytes must *not* overlap with the region of memory
 ///   beginning at `dst` with the same size.
 ///
+/// * For every uninitialized byte in `src`, the corresponding byte in `dst` will also be
+///   uninitialized; using copied uninitialized bytes is undefined behavior.
```

Comment:
The part after the `;` is odd, and also not entirely true depending on what exactly you mean by "using".

This is not really part of the safety contract of this function but part of its operational sepcification, so this is the wrong part of the docs to talk about it. It should go up in the first section, I think.

Reply:
Fair enough.

If we add a "what are uninit bytes" section [as I suggested](https://github.com/rust-lang/rust/pull/67826#discussion_r363226160), maybe this could go up top, and then link to that section to explain it more fully?